### PR TITLE
Multiple Keymap Updates - Unikorn/Curiosity/Prime_E/Ferris/Corne

### PR DESCRIPTION
**Pull request by sidcarter**
## Description

Updates to multiple keymaps for multiple keyboards:

* Ferris
* Curiosity
* WT 60-D
* Prime_E
* Corne

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/cheshire/curiosity/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/*
```
Comment (drashna) [inline keyboards/cheshire/curiosity/keymaps/via/keymap.c:1]: This should be in a separate PR, ideally.

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 2]
```diff
@@ -1,114 +1,111 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
 #endif
```
Comment (drashna) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:25]: Please remove. This shouldn't be needed.
```suggestion
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 3]
```diff
@@ -1,114 +1,111 @@
...
-    _ADJUST,
-};
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 3
```
Comment (drashna) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:36]: Please revert. Enums are the preferred method for labeling layers, as they're easier to maintain long term and less prone to weird issues. 

https://docs.qmk.fm/#/pr_checklist?id=keymap-prs

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 4]
```diff
@@ -1,114 +1,111 @@
...
-    return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
-}
-
 int RGB_current_mode;
 
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
```
Comment (drashna) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:108]: Please remove this function.

It's been deprecated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 5]
```diff
@@ -120,7 +117,7 @@ void update_tri_layer_RGB(uint8_t layer1, uint8_t layer2, uint8_t layer3) {
 
 void matrix_init_user(void) {
     #ifdef RGBLIGHT_ENABLE
-      RGB_current_mode = rgblight_get_mode();
+      RGB_current_mode = rgblight_config.mode;
```
Comment (drashna) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:120]: Please revert.  The preferred method is to use the query function.
```suggestion
      RGB_current_mode = rgblight_get_mode();
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 6]
```diff
@@ -185,6 +182,11 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
   }
 
   switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        persistent_default_layer_set(1UL<<_QWERTY);
+      }
+      return false;
```
Comment (drashna) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:189]:
```suggestion
    case QWERTY:
      if (record->event.pressed) {
       set_single_persistent_default_layer(_QWERTY);
      }
      break;
```

Also, because you have no other default layers, this is completely unneeded and could be safely removed.

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 7]
```diff
@@ -210,6 +212,24 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
           layer_off(_ADJUST);
         }
         return false;
+    case RGB_MOD:
+      #ifdef RGBLIGHT_ENABLE
+        if (record->event.pressed) {
+          rgblight_mode(RGB_current_mode);
+          rgblight_step();
+          RGB_current_mode = rgblight_config.mode;
```
Comment (drashna) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:220]:
```suggestion
          RGB_current_mode = rgblight_get_mode();
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 8]
```diff
@@ -210,6 +212,24 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
...
+        }
+      #endif
+      return false;
+    case RGBRST:
+      #ifdef RGBLIGHT_ENABLE
+        if (record->event.pressed) {
+          eeconfig_update_rgblight_default();
+          rgblight_enable();
+          RGB_current_mode = rgblight_config.mode;
```
Comment (drashna) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:229]:
```suggestion
          RGB_current_mode = rgblight_get_mode();
```

File: keyboards/ferris/keymaps/madhatter/rules.mk [hunk 9]
```diff
@@ -0,0 +1,2 @@
+BOOTMAGIC_ENABLE = yes
```
Comment (drashna) [inline keyboards/ferris/keymaps/madhatter/rules.mk:1]: This should be set to full or lite, ideally. 

And "lite" is the preferred option, as it's less prone to weird issues.

File: keyboards/primekb/prime_e/keymaps/madhatter/keymap.c [hunk 10]
```diff
@@ -96,17 +85,17 @@ void matrix_scan_user(void) {
 
 //function for layer indicator LED
 layer_state_t layer_state_set_user(layer_state_t state) {
-    if (layer_state_cmp(state, 1)) {
+    if (layer_state_cmp(state, 0)) {
         writePinHigh(CAPS_LED_PIN);
     } else {
         writePinLow(CAPS_LED_PIN);
     }
```
Comment (drashna) [inline keyboards/primekb/prime_e/keymaps/madhatter/keymap.c:92]: This could be simplified to: 
```suggestion
    writePin(CAPS_LED_PIN, layer_state_cmp(state, 0));
```

Also, layer 0 may show up as always on, due to how it's handled.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/ferris/keymaps/madhatter/rules.mk (hunk 9 above)
Comment (sidcarter) [reply keyboards/ferris/keymaps/madhatter/rules.mk]: done

File: keyboards/crkbd/keymaps/madhatter/keymap.c (hunk 4 above)
Comment (sidcarter) [reply keyboards/crkbd/keymaps/madhatter/keymap.c]: removed

File: keyboards/crkbd/keymaps/madhatter/keymap.c (hunk 3 above)
Comment (sidcarter) [reply keyboards/crkbd/keymaps/madhatter/keymap.c]: fixed

File: keyboards/cheshire/curiosity/keymaps/via/keymap.c (hunk 1 above)
Comment (sidcarter) [reply keyboards/cheshire/curiosity/keymaps/via/keymap.c:1]: I'm hoping this can proceed for now. will remember it for next time

File: keyboards/primekb/prime_e/keymaps/madhatter/keymap.c [hunk 11]
```diff
@@ -96,17 +85,13 @@ void matrix_scan_user(void) {
 
 //function for layer indicator LED
 layer_state_t layer_state_set_user(layer_state_t state) {
+    writePin(CAPS_LED_PIN, layer_state_cmp(state, 0));
     if (layer_state_cmp(state, 1)) {
```
Comment (drashna) [inline keyboards/primekb/prime_e/keymaps/madhatter/keymap.c:89]: The same can be done here, and for layer 2, as well.

File: keyboards/ferris/keymaps/madhatter/rules.mk (hunk 9 above)
Comment (sidcarter) [reply keyboards/ferris/keymaps/madhatter/rules.mk]: made it lite

File: keyboards/crkbd/keymaps/madhatter/keymap.c (hunk 3 above)
Comment (sidcarter) [reply keyboards/crkbd/keymaps/madhatter/keymap.c]: done

File: keyboards/crkbd/keymaps/madhatter/keymap.c (hunk 4 above)
Comment (sidcarter) [reply keyboards/crkbd/keymaps/madhatter/keymap.c]: done

File: keyboards/primekb/prime_e/keymaps/madhatter/keymap.c (hunk 11 above)
Comment (sidcarter) [reply keyboards/primekb/prime_e/keymaps/madhatter/keymap.c:89]: done

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 12]
```diff
@@ -1,114 +1,100 @@
...
-#define LY_RSE MO(_RAISE)
-
-#define KY_CESC LCTL_T(KC_ESC)
+enum macro_keycodes {
+  KC_SAMPLEMACRO,
+};
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
-  [_QWERTY] = LAYOUT(
+  [_QWERTY] = LAYOUT_split_3x6_3( \
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:44]:
```suggestion
  [_QWERTY] = LAYOUT_split_3x6_3(
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 13]
```diff
@@ -1,114 +1,100 @@
...
   //,-----------------------------------------------------.                    ,-----------------------------------------------------.
-       KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,                         KC_Y,    KC_U,    KC_I,    KC_O,   KC_P,  KC_BSPC,
+       KC_TAB,    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,                         KC_Y,    KC_U,    KC_I,    KC_O,   KC_P,  KC_BSPC,\
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:46]:
```suggestion
       KC_TAB,    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,                         KC_Y,    KC_U,    KC_I,    KC_O,   KC_P,  KC_BSPC,
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 14]
```diff
@@ -1,114 +1,100 @@
...
   //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
-      KC_LCTL,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,                         KC_H,    KC_J,    KC_K,    KC_L, KC_SCLN, KC_QUOT,
+      KC_LCTL,    KC_A,    KC_S,    KC_D,    KC_F,    KC_G,                         KC_H,    KC_J,    KC_K,    KC_L, KC_SCLN, KC_QUOT,\
   //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
-      KC_LSFT,  KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                         KC_N,    KC_M, KC_COMM,  KC_DOT, KC_SLSH, KC_ENT,
+      KC_LSFT,    KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                         KC_N,    KC_M, KC_COMM,  KC_DOT, KC_SLSH, KC_ENT ,\
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:50]:
```suggestion
      KC_LSFT,    KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                         KC_N,    KC_M, KC_COMM,  KC_DOT, KC_SLSH, KC_ENT ,
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 15]
```diff
@@ -1,114 +1,100 @@
...
   //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
-                                          KC_LALT,  LY_LWR, KC_LCMD,     KC_SPC,  LY_RSE,  KC_ENT
+                                          KC_LCTL,  LOWER, KC_BSPC,     KC_SPC,  RAISE,  KC_RCMD \
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:52]:
```suggestion
                                          KC_LCTL,  LOWER, KC_BSPC,     KC_SPC,  RAISE,  KC_RCMD
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 16]
```diff
@@ -1,114 +1,100 @@
...
                                       //`--------------------------'  `--------------------------'
 
   ),
 
-  [_LOWER] = LAYOUT(
+  [_LOWER] = LAYOUT_split_3x6_3( \
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:57]:
```suggestion
  [_LOWER] = LAYOUT_split_3x6_3(
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 17]
```diff
@@ -1,114 +1,100 @@
...
   //,-----------------------------------------------------.                    ,-----------------------------------------------------.
-       KC_ESC, KC_EXLM,   KC_AT, KC_HASH,  KC_DLR, KC_PERC,                      KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_DEL,
+       KC_ESC,    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                         KC_6,    KC_7,    KC_8,    KC_9,    KC_0, KC_BSPC,\
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:59]:
```suggestion
       KC_ESC,    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                         KC_6,    KC_7,    KC_8,    KC_9,    KC_0, KC_BSPC,
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 18]
```diff
@@ -1,114 +1,100 @@
...
   //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
-      KC_LCTL, XXXXXXX, KC_LEFT, KC_DOWN,   KC_UP,KC_RIGHT,                      KC_MINS,  KC_EQL, KC_LCBR, KC_RCBR, KC_PIPE, KC_GRV,
+      KC_LCTL, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                      KC_LEFT, KC_DOWN,   KC_UP,KC_RIGHT, XXXXXXX, XXXXXXX,\
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:61]:
```suggestion
      KC_LCTL, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                      KC_LEFT, KC_DOWN,   KC_UP,KC_RIGHT, XXXXXXX, XXXXXXX,
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 19]
```diff
@@ -1,114 +1,100 @@
...
   //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
-      KC_LSFT, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                      KC_UNDS, KC_PLUS, KC_LBRC, KC_RBRC, KC_BSLS, KC_TILD,
+      KC_LSFT, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,\
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:63]:
```suggestion
      KC_LSFT, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 20]
```diff
@@ -1,114 +1,100 @@
...
   //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
-                                          KC_LALT,  LY_LWR, KC_LCMD,     KC_SPC,  LY_RSE, KC_ENT
+                                          KC_TRNS,  LOWER, KC_TRNS,    KC_TRNS,   RAISE, KC_TRNS \
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:65]:
```suggestion
                                          KC_TRNS,  LOWER, KC_TRNS,    KC_TRNS,   RAISE, KC_TRNS
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 21]
```diff
@@ -1,114 +1,100 @@
...
                                       //`--------------------------'  `--------------------------'
-  ),
+    ),
 
-  [_RAISE] = LAYOUT(
+  [_RAISE] = LAYOUT_split_3x6_3( \
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:69]:
```suggestion
  [_RAISE] = LAYOUT_split_3x6_3(
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 22]
```diff
@@ -1,114 +1,100 @@
...
   //,-----------------------------------------------------.                    ,-----------------------------------------------------.
-       KC_ESC,    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                         KC_6,    KC_7,    KC_8,    KC_9,    KC_0, KC_DEL,
+       KC_ESC, KC_EXLM,   KC_AT, KC_HASH,  KC_DLR, KC_PERC,                      KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_BSPC,\
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:71]:
```suggestion
       KC_ESC, KC_EXLM,   KC_AT, KC_HASH,  KC_DLR, KC_PERC,                      KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_BSPC,
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 23]
```diff
@@ -1,114 +1,100 @@
...
   //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
-      KC_LCTL, XXXXXXX, KC_PGUP, KC_HOME,   KC_END,KC_PGDN,                      KC_MINS,  KC_EQL, KC_LCBR, KC_RCBR, KC_PIPE, KC_GRV,
+      KC_LCTL, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                      KC_MINS,  KC_EQL, KC_LCBR, KC_RCBR, KC_PIPE,  KC_GRV,\
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:73]:
```suggestion
      KC_LCTL, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                      KC_MINS,  KC_EQL, KC_LCBR, KC_RCBR, KC_PIPE,  KC_GRV,
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 24]
```diff
@@ -1,114 +1,100 @@
...
   //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
-      KC_LSFT, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                      KC_UNDS, KC_PLUS, KC_LBRC, KC_RBRC, KC_BSLS, KC_TILD,
+      KC_LSFT, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                      KC_UNDS, KC_PLUS, KC_LBRC, KC_RBRC, KC_BSLS, KC_TILD,\
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:75]:
```suggestion
      KC_LSFT, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                      KC_UNDS, KC_PLUS, KC_LBRC, KC_RBRC, KC_BSLS, KC_TILD,
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 25]
```diff
@@ -1,114 +1,100 @@
...
   //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
-                                          KC_LALT,  LY_LWR, KC_LCMD,     KC_SPC,  LY_RSE, KC_ENT
+                                          KC_TRNS,  LOWER, KC_BSPC,    KC_TRNS,   RAISE, KC_TRNS \
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:77]:
```suggestion
                                          KC_TRNS,  LOWER, KC_BSPC,    KC_TRNS,   RAISE, KC_TRNS
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 26]
```diff
@@ -1,114 +1,100 @@
...
                                       //`--------------------------'  `--------------------------'
-    ),
+  ),
 
-  [_ADJUST] = LAYOUT(
+  [_ADJUST] = LAYOUT_split_3x6_3( \
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:81]:
```suggestion
  [_ADJUST] = LAYOUT_split_3x6_3(
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 27]
```diff
@@ -1,114 +1,100 @@
...
   //,-----------------------------------------------------.                    ,-----------------------------------------------------.
-        RESET, KC_BTN1, KC_MS_U, KC_BTN2, XXXXXXX, XXXXXXX,                      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+      XXXXXXX,  RGBRST, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,\
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:83]:
```suggestion
      XXXXXXX,  RGBRST, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 28]
```diff
@@ -1,114 +1,100 @@
...
   //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
-      XXXXXXX, KC_MS_L, KC_MS_D, KC_MS_R, XXXXXXX, XXXXXXX,                      RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI, XXXXXXX, XXXXXXX,
+      RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI, XXXXXXX, XXXXXXX,                      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,\
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:85]:
```suggestion
      RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI, XXXXXXX, XXXXXXX,                      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 29]
```diff
@@ -1,114 +1,100 @@
...
   //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
-      XXXXXXX,  RGBRST, EEP_RST, AG_TOGG, XXXXXXX, XXXXXXX,                      RGB_MOD, RGB_HUD, RGB_SAD, RGB_VAD, XXXXXXX, XXXXXXX,
+      RGB_MOD, RGB_HUD, RGB_SAD, RGB_VAD, XXXXXXX, XXXXXXX,                      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,   RESET,\
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:87]:
```suggestion
      RGB_MOD, RGB_HUD, RGB_SAD, RGB_VAD, XXXXXXX, XXXXXXX,                      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,   RESET,
```

File: keyboards/crkbd/keymaps/madhatter/keymap.c [hunk 30]
```diff
@@ -1,114 +1,100 @@
...
   //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
-                                          KC_LALT,  LY_LWR, KC_LCMD,     KC_SPC,  LY_RSE, KC_ENT
+                                          KC_TRNS,  LOWER, KC_TRNS,    KC_TRNS,   RAISE, KC_TRNS \
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/madhatter/keymap.c:89]:
```suggestion
                                          KC_TRNS,  LOWER, KC_TRNS,    KC_TRNS,   RAISE, KC_TRNS
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]
Oops, still some outstanding stuff.

**Comment by sidcarter** [comment]
what else outstanding @fauxpark ?

**Comment by fauxpark** [comment]
image
(I really don't like how GitHub does that)

**Comment by sidcarter** [comment]
lol, you're right. and every accept suggestion then just resets the convo to load more. lol

@fauxpark fixed now

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]
